Docs/cassandra loader timestamp (#4016)
yugabyte/yugabyte-db: docs/content/latest/manage/data-migration/ycql/bulk-export.md

---
title: Bulk export
linkTitle: Bulk export
description: Bulk export
image: /images/section_icons/manage/enterprise.png
headcontent: Bulk export data from YugabyteDB.
menu:
  latest:
    identifier: manage-bulk-export
    parent: manage-bulk-import-export
    weight: 707
aliases:
  - /latest/manage/data-migration/bulk-export/
isTocNested: true
showAsideToc: true
---

<ul class="nav nav-tabs-alt nav-tabs-yb">
  <li >
    <a href="" class="nav-link active">
      <i class="icon-cassandra" aria-hidden="true"></i>
      YCQL
    </a>
  </li>
</ul>

> This page documents bulk export for YugabyteDB’s [Cassandra compatible YCQL API](../../../api/ycql). To export data from a YugabyteDB (or even an Apache Cassandra) table, you can use the [`cassandra-unloader`](https://github.com/yugabyte/cassandra-loader#cassandra-unloader) tool.

Reviewer: ```suggestion
This page documents bulk export for YugabyteDB’s [Cassandra-compatible YCQL API](../../../api/ycql). To export data from a YugabyteDB (or an Apache Cassandra) table, you can use the [`cassandra-unloader`](https://github.com/yugabyte/cassandra-loader#cassandra-unloader) tool.
```

```

For additional options to cassandra-unloader, see [here](https://github.com/yugabyte/cassandra-loader#cassandra-unloader).

{{< note title="Always specify timezone" >}}

> `cassnadra-loader` & `cassandra-unloader` don't add the timezone on their default timestamp format.

Reviewer 2: spelling issue - cassndara

> `cassandra-loader` & `cassandra-unloader` don't add the timezone on their default timestamp format.

Reviewer: ```suggestion
The time zone is not added to the default timestamp formats when using the `cassandra-loader` and `cassandra-unloader` utilities.

> Make sure that timestamps are exported/imported with the same format and including the timezone.

Reviewer: ```suggestion
Make sure that timestamps are exported and imported in the same format, including the time zone.
```

Examples:  `yyyy-MM-dd HH:mm:ss.SSSZ` and `yyyy-MM-dd HH:mm:ss.SSSXXX`.
{{< /note >}}

{{< note title="Use tab character for delim on JSONB columns" >}}

> The default delim `,` doesn't work with `JSONB` columns. Use tab character for delim `-delim $'\t'`.

Reviewer: ```suggestion
The default delimiter (`,`) does not work with `JSONB` columns. Use the tab character for delim `-delim $'\t'`.
```
